Prog/Housing_needs_units_targets.sas

```diff
@@ -0,0 +1,859 @@
+	    loan = .9 * valueh_a;
+	    month_mortgage= (3.69 / 12) / 100; 
+	    monthly_PI = loan * month_mortgage * ((1+month_mortgage)**360)/(((1+month_mortgage)**360)-1);
+
+	    ****
+	    Calculate PMI and taxes/insurance to add to Monthly_PI to find total monthly payment
+	    ******;
+	    
+	    PMI = (.007 * loan ) / 12; **typical annual PMI is .007 of loan amount;
+	    tax_ins = .25 * monthly_PI; **taxes assumed to be 25% of monthly PI; 
+	    total_month = monthly_PI + PMI + tax_ins; **Sum of monthly payment components;
+		
+			/*create owner cost level categories*/ 
+			ownlevel=.;
+				if 0 <=total_month<1200 then ownlevel=1;
+				if 1200 <=total_month<1800 then ownlevel=2;
+				if 1800 <=total_month<2500 then ownlevel=3;
+				if 2500 <=total_month<3200 then ownlevel=4;
+				if 3200 <=total_month<4200 then ownlevel=5;
+				if total_month >= 4200 then ownlevel=6;
+			
+			/*create  categories now used in targets for renter/owner costs combined*/ 
+				allcostlevel=.;
+				if total_month<800 then allcostlevel=1;
+				if 800 <=total_month<1300 then allcostlevel=2;
+				if 1300 <=total_month<1800 then allcostlevel=3;
+				if 1800 <=total_month<2500 then allcostlevel=4;
+				if 2500 <=total_month<3500 then allcostlevel=5;
+				if total_month >= 3500 then allcostlevel=6; 
+
+
+	  end;
+
+		label rentlevel = 'Rent Level Categories based on Current Gross Rent'
+		 		  allcostlevel='Housing Cost Categories (tenure combined) based on Current Rent or First-time Buyer Mtg'
+				  ownlevel = 'Owner Cost Categories based on First-Time HomeBuyer Costs'
+				;
+	format ownlevel ocost. rentlevel rcost. vacancy_r VACANCY_F. allcostlevel acost. ; 
+
+	*output other vacant - seasonal separately ;
+	if vacancy in (1, 2, 3) then output Housing_needs_vacant_&year.;
+	else if vacancy in (4, 7, 9) then output other_vacant_&year.; 
+	run;
+
+%mend single_year; 
+
+%single_year(2013);
+%single_year(2014);
+%single_year(2015); 
+%single_year(2016);
+%single_year(2017);
+
+/*merge single year data*/ 
+data fiveyeartotal;
+	set Housing_needs_baseline_2013 Housing_needs_baseline_2014 Housing_needs_baseline_2015 Housing_needs_baseline_2016 Housing_needs_baseline_2017;
+
+hhwt_5=hhwt*.2; 
+
+run; 
+proc means data= fiveyeartotal;
+class hud_inc;
+var Costratio ;
+weight hhwt_5;
+run;
+data fiveyeartotal_vacant;
+	set Housing_needs_vacant_2013 Housing_needs_vacant_2014 Housing_needs_vacant_2015 Housing_needs_vacant_2016 Housing_needs_vacant_2017;
+
+hhwt_5=hhwt*.2; 
+
+run; 
+data fiveyeartotal_othervacant;
+   set other_vacant_2013 other_vacant_2014 other_vacant_2015 other_vacant_2016 other_vacant_2017;
+
+hhwt_5=hhwt*.2;
+
+run; 
+proc sort data =fiveyeartotal_othervacant;
+by jurisdiction;
+proc freq data=fiveyeartotal_othervacant;
+by jurisdiction;
+tables vacancy /nopercent norow nocol out=other_vacant;
+weight hhwt_5;
+format jurisdiction jurisdiction.;
+run; 
+proc export data=other_vacant
```

Comment:
Here again, it would just be helpful to have a short comment about what this output is meant to represent.